.github/actions/setupGitForOSBotify/action.yml

```diff
@@ -0,0 +1,64 @@
+# This action configures Git for OSBotify.
+#
+# By default, the token it provides is a Github App for Github Authentication.
+# GitHub Apps have higher rate limits.
+#
+# If you only need a local git setup, you can skip that step with SETUP_AS_APP: false
+
+name: Setup Git for OSBotify
+description: Setup Git for OSBotify
+
+inputs:
+  OP_SERVICE_ACCOUNT_TOKEN:
```

Comment:
Just want to make sure this will be secured with the `***`?

Reply:
We chatted about this a bit more, but we pass secrets to actions all the time and they remain obfuscated. Seems like this should be the same